errors.go

```diff
@@ -260,19 +260,22 @@ func (fe *fieldError) Error() string {
 // NOTE: if no registered translation can be found, it returns the original
 // untranslated error message.
 func (fe *fieldError) Translate(ut ut.Translator) string {
-
+	var (
+		fn TranslationFunc
+		ok bool
+	)
+	
 	m, ok := fe.v.transTagFunc[ut]
 	if !ok {
 		return fe.Error()
 	}
 
-	fn, ok := m[fe.tag]
+	fn, ok = m[fe.tag]
 	if !ok {
 		fn, ok = m[fe.actualTag]
 		if !ok {
 			return fe.Error()
 		}
-		return fe.Error()
```

Comment:
Looks like this is the only line that needs to change ?

Reply:
You need to check variable `m` with 2 different tags `fe.tag` and `fe.actualTag`

Reply:
Ah, correct, got confused when looking at the diffs, with 1 commit it is clearer now